File: semester4/web_programming/exam/MoreExams/Topics ASP/Topics/Models/Post.cs
using System;
using System.Collections.Generic;

namespace Topics.Models
{
    public partial class Post
    {
        public int Id { get; set; }
        public string User { get; set; }
        public int Topicid { get; set; }
        public string Text { get; set; }
        public int Date { get; set; }

        public override string ToString()
        {
            return $"{nameof(User)}: {User}, {nameof(Topicid)}: {Topicid}, {nameof(Text)}: {Text}, {nameof(Date)}: {Date}";
        }
    }
}

File: semester4/web_programming/exam/MoreExams/Topics JSP/src/main/java/ro/ubb/topics/model/Post.java
package ro.ubb.topics.model;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Post extends BaseEntity<Long>{

    String username;

    int topicid;

    String text;

    int date;
}
